scripts/kind-e2e/e2e.sh

```diff
@@ -459,17 +459,20 @@ elif [[ $5 = helm ]]; then
     helm=true
     setup_cluster2_gateway
     setup_cluster3_gateway
-#    for i in 2 3; do
-#      context=cluster$i
-#      kubectl config use-context $context
-#
-#      verify_subm_engine_pod
-#      verify_subm_routeagent_pod
-#      verify_subm_engine_container
-#      verify_subm_routeagent_container
-#      verify_subm_engine_secrets
-#      verify_subm_routeagent_secrets
-#    done
+    for i in 2 3; do
+      context=cluster$i
+      kubectl config use-context $context
+
+      # The Helm deploy doesn't respect namespace config, hardcode to what it uses
```

Comment:
Well, the fact is that more than helm respecting, we may make the operator deploy on the submariner namespace instead, not the operator namespace :)

Reply:
Yes, we need to support installing the operator in a different namespace.

Reply:
I'm not sure I understand what you guys mean. The Operator *does* respect this flag, *does* deploy into whatever namespace. It's the default Helm-based deployments that will always deploy into "submariner" namespace, without regard for this flag, causing the tests to (correctly) fail. This just hard-codes the NS for Helm-based deploys to pass their limitation, is not relevant to Operator-based deploys.

Reply:
Indeed, sorry, I got the configurations mixed up!